### ISSUE TITLE
Scanned shows look nothing like screenshots

### ISSUE THREAD

**Issue by dmanlfc** [artemanufrij/playmyvideos#12]
I have my TV Shows set out like so:

<Show Name>/<Season>/<Episode Name>

i.e. The Orville/Season 1/The Orville - 1x01 - Old Wounds.mkv

The Outcome looks nothing like the Screenshots. Are you doing any TVDB scraping yet?
All Season1 folders regardless of the parent directory get bundled together
Can you have a bit more logic when importing shows Artem?

Cheers

**Comment by artemanufrij** [comment]
@dmanlfc try following: 
The Orville/**The Orville -** Season 1/The Orville - 1x01 - Old Wounds.mkv

**Comment by dmanlfc** [comment]
Well that will work but I don't want to rename all my subdirectories.
I use this file share for Plex - thought I'd give your new creation a try,
Have you thought about using TheTVDB for TV Show scraping? https://api.thetvdb.com/swagger

I don't even have to have my files names tidy & Plex will scrape the content accordingly provided it can determine the Series & Episode number also.

**Comment by artemanufrij** [comment]
@dmanlfc Hm... I will check it... give me some time

**Comment by dmanlfc** [comment]
No worries brother - see what you think.
You can grab banner & poster art too.

Here is an API source: https://github.com/dbr/tvdb_api
Example program using it: https://github.com/dbr/tvnamer

**Comment by dmanlfc** [comment]
Java wrapper - easier to convert to Vala?

https://github.com/UweTrottmann/thetvdb-java/blob/master/src/main/java/com/uwetrottmann/thetvdb/TheTvdb.java

**Comment by artemanufrij** [comment]
@dmanlfc check current master branch please - you should rescan your content:
![bildschirmfoto von 2018-01-16 19 19 53](https://user-images.githubusercontent.com/16517095/35004987-425f2508-faf2-11e7-82c5-94c57e75fb16.png)

**Comment by dmanlfc** [comment]
Yes works better - now the shows can be easier to find & seperated from the rest.
It would be better however if instead of having Banshee - Season 1,2,3 & 4 as separate thumbs on the main screen you'll only see Banshee - once clicked on it, you go into a separate sub view showing all the seasons you have with scraped Season poster thumbs & a scraped Banner.
Once you click on the Season the Season playlist pops out from the right with only 'Episode' names not the full file name.
